api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -66,9 +66,14 @@ type imageBuilder struct {
 }
 
 const (
-	containerName      = "pyfunc-image-builder"
-	kanikoSecretName   = "kaniko-secret"
-	tickDurationSecond = 5
+	containerName    = "pyfunc-image-builder"
+	kanikoSecretName = "kaniko-secret"
+	// jobDeletionTickDurationMilliseconds is the interval at which the API server checks if a job has completed
+	jobCompletedTickDurationSecond = 5
+	// jobDeletionTimeoutSeconds is the maximum time to wait for a job to be deleted from a cluster
+	jobDeletionTimeoutSeconds = 10
```

Comment:
Is this sufficient? Or should we make it 30 seconds, to give ample time, just in case?

Reply:
+1 on making it 30 seconds